sound/soc/sof/sof-priv.h

```diff
@@ -292,7 +395,9 @@ struct snd_sof_dev {
 	spinlock_t ipc_lock;	/* lock for IPC users */
 	spinlock_t hw_lock;	/* lock for HW IO access */
 	struct pci_dev *pci;
-
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_VIRTIO_FE)
```

Comment:
Here I think we may need it, because in FE, the vdev is as type  virtio_device which is standard virtio device, but in BE it is as virtio_dev_info in the VBS, this can be changed when we make the FE/BE driver together, and if VBS is not used.